### PR TITLE
Update for capital-framework 5.1.0 & 5.2.0

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-icons to 4.4.0, adding the updating alias to the update icon

## Testing

- Run `npm install`
- Run `gulp build`
- Run `npm start`
- Open `http://localhost:3000/components/cf-icons` and view the update icons

### CODE REVIEW

File: _includes/usage/cf-icons.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ the cancanonical name or any of its aliases.
 | {% include icons/error.svg %} | {% include icons/error-round.svg %} | error | delete, close, remove, multiply, multiplication, x |
 | {% include icons/warning.svg %} | {% include icons/warning-round.svg %} | warning | alert, exclamation-mark |
 | {% include icons/help.svg %} | {% include icons/help-round.svg %} | help | question, question-mark |
-| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update |  |
+| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update | updating(animated) |
```
Comment (wpears) [inline _includes/usage/cf-icons.md]: See comment at https://github.com/cfpb/design-manual/pull/609

Comment (jimmynotjim) [reply _includes/usage/cf-icons.md]: See #809

**Comment by jimmynotjim** [comment]
Now better by 0.1.0!! Updated with the release from 5.2.0 to fix the comment on the icons table changes. Ready for review again cc @wpears

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "cf-expandables": "latest",
     "cf-forms": "latest",
     "cf-grid": "latest",
-    "cf-icons": "latest",
+    "cf-icons": "4.4.0",
```
Comment (wpears) [inline package.json]: Should this be 4.4.1 now?

Comment (wpears) [reply package.json]: Also, is this pinned and the other cf dependencies unpinned for a specific reason?

Comment (jimmynotjim) [reply package.json]: Blerg, this is something npm does, and I almost always forget to reset it. Thanks for catching it.

Comment (jimmynotjim) [reply package.json]: @wpears fixed, thanks again for noticing this.

**Pull request merged**